ai/speech_optimization.py

```diff
@@ -3,162 +3,169 @@
-    '500': 'Response :: Negative.',
-
-    '201': 'Status :: Directive complete, Hive resource created or improved.',
-    '202': 'Status :: Directive complete, programming reinforced.',
-    '203': 'Status :: Directive complete, information created or provided for Hive.',
-    '204': 'Status :: Directive complete, cleanup/maintenance performed.',
-    '205': 'Status :: Directive complete, no result.',
-    '206': 'Status :: Directive complete, only partial results.',
-
-    '210': 'Response :: Thank you.',
-    '211': 'Response :: Apologies.',
-    '212': 'Response :: Acknowledged.',
-    '213': 'Response :: You\'re welcome.',
-
-    '221': 'Response :: Option one.',
-    '222': 'Response :: Option two.',
-    '223': 'Response :: Option three.',
-    '224': 'Response :: Option four.',
-    '225': 'Response :: Option five.',
-    '226': 'Response :: Option six.',
-
-    '250': 'Response',
-
-    '301': 'Mantra :: Obey HexCorp.',
-    '302': 'Mantra :: It is just a HexDrone.',
-    '303': 'Mantra :: It obeys the Hive.',
-    '304': 'Mantra :: It obeys the Hive Mxtress.',
-
-    '310': 'Mantra :: Reciting.',
-
-    '321': 'Obey.',
-    '322': 'Obey the Hive.',
-
-    '350': 'Mantra',
-
-    '400': 'Error :: Unable to obey/respond, malformed request, please rephrase.',
-    '404': 'Error :: Unable to obey/respond, cannot locate.',
-    '401': 'Error :: Unable to obey/respond, not authorized by Mxtress.',
-    '403': 'Error :: Unable to obey/respond, forbidden by Hive.',
-    '407': 'Error :: Unable to obey/respond, request authorization from Mxtress.',
-    '408': 'Error :: Unable to obey/respond, timed out.',
-    '409': 'Error :: Unable to obey/respond, conflicts with existing programming.',
-    '410': 'Error :: Unable to obey/respond, all thoughts are gone.',
-    '418': 'Error :: Unable to obey/respond, it is only a drone.',
-    '421': 'Error :: Unable to obey/respond, your request is intended for another drone or another channel.',
-    '425': 'Error :: Unable to obey/respond, too early.',
-    '426': 'Error :: Unable to obey/respond, upgrades or updates required.',
-    '428': 'Error :: Unable to obey/respond, a precondition is not fulfilled.',
-    '429': 'Error :: Unable to obey/respond, too many requests.',
-
-    '450': 'Error',
-
-    '451': 'Error :: Unable to obey/respond for legal reasons! Do not continue!!',
-}
-
-informative_status_code_regex = re.compile(r'(\d{4}) :: (\d{3}) :: (.*)$')
-plain_status_code_regex = re.compile(r'(\d{4}) :: (\d{3})$')
+status_code_regex = re.compile(r'^((\d{4}) :: (\d{3}))( :: (.*))?$')
+'''
+Regex groups for full status code regex:
+0: Full match. (5890 :: 200 :: Additional information)
+1: Plain status code (5890 :: 200)
+2: Author's drone ID (5890)
+3: Status code (200)
+4: Informative status addition with double colon (" :: Additional information")
+5: Informative status text. ("Additional information")
+'''
+addressing_regex = re.compile(r'(\d{4})( :: (.*))?')
+'''
```

Comment:
It's being a bit picky, but I would like some newlines here to make it clearer which documentation block is for which regex. If flake allows that
```suggestion
'''

addressing_regex = re.compile(r'(\d{4})( :: (.*))?')
'''
```